packages/user_packages/iotcc_mqtt_dev.py

```diff
@@ -0,0 +1,186 @@
+# -*- coding: utf-8 -*-
+# ----------------------------------------------------------------------------#
+#  Copyright © 2015-2016 VMware, Inc. All Rights Reserved.                    #
```

Comment:
What is this dev file all about. Same as iotcc_mqtt.py?

Reply:
Ok I see this is the child package that was being used and renamed. I don't understand the naming here though

Reply:
If this is a new file should the Copyright be 2018-2019?

Reply:
This is the license file which was provided to us during the inception of Liota hence I`m not sure if it should be updated.

Renamed to iotcc_mqtt_device.py